java/test/jmri/implementation/PackageTest.java

```diff
@@ -47,15 +45,18 @@ public static Test suite() {
         suite.addTest(OffsetHighCvProgrammerFacadeTest.suite());
         suite.addTest(ResettingOffsetHighCvProgrammerFacadeTest.suite());
         suite.addTest(RouteTest.suite());
-        suite.addTest(SE8cSignalHeadTest.suite());
-        suite.addTest(SignalHeadSignalMastTest.suite());
         suite.addTest(SignalSpeedMapTest.suite());
         suite.addTest(SignalSystemFileCheckTest.suite());
         suite.addTest(SingleTurnoutSignalHeadTest.suite());
         suite.addTest(TwoIndexTcsProgrammerFacadeTest.suite());
         suite.addTest(BundleTest.suite());
+        // Converted to JUnit4:
         suite.addTest(new junit.framework.JUnit4TestAdapter(DccConsistTest.class));
+        suite.addTest(new junit.framework.JUnit4TestAdapter(DccSignalHeadTest.class));
```

Comment:
Do you mean we should not put them (in roughly the same roder) in a separate block "Converted to JUnit4" at the bottom, but replace the suite.addTest() lines in the existing list instead? Within this block, I kept the order identical as I had read a discussion about this being relevant.